conftest.py

```diff
@@ -1,9 +1,18 @@
 # content of conftest.py
 import sys
+import test_fixtures
+import pytest
```

Comment:
Order imports per style guide: https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Imports_formatting#Imports_formatting

So:

    import sys
    
    import pytest
    
    import test_fixtures

I would put spaces between them, as each import is a different category of module.